src/index.ts

```diff
@@ -31,8 +31,8 @@ export function useZustand<State, Slice>(
     () => [slice, store],
   );
   useEffect(() => {
-    const unsubscribe = store.subscribe(rerender as DispatchWithoutAction);
-    (rerender as DispatchWithoutAction)();
+    const unsubscribe = store.subscribe(() => rerender(false));
+    rerender(false);
```

Comment:
```suggestion
    const unsubscribe = store.subscribe(() => (rerender as DispatchWithoutAction)());
    (rerender as DispatchWithoutAction)();
```
is smaller in bundle.

Reply:
I see. I was wondering why you weren't making it more readable.